README.md

```diff
@@ -112,49 +113,112 @@ ebcli rest-server --trust-node
+
+# Start the Bridge's blockchain
+$ ebd start
+```
+
+### Terminal 4: Start the Relayer service
 
-For automated relaying, there is a relayer service that can be run that will automatically watch and relay events.
+For automated relaying, there is a relayer service that can be run that will automatically watch and relay events (local web socket and deployed address parameters may vary).
 
 ```
 # Check ebrelayer connection to ebd
 ebrelayer status
 
-# Initialize the Relayer service for automatic claim processing
-ebrelayer init testing wss://ropsten.infura.io/ws ec6df30846baab06fce9b1721608853193913c19 "LogLock\(bytes32,address,bytes,address,uint256,uint256\)" validator
+# Start ebrelayer on the contract's deployed address with [LOCAL_WEB_SOCKET] and [PEGGY_DEPLOYED_ADDRESS]
+# Example [LOCAL_WEB_SOCKET]: ws://127.0.0.1:8545/
+# Example [PEGGY_DEPLOYED_ADDRESS]: 0xC4cE93a5699c68241fc2fB503Fb0f21724A624BB
+
+$ ebrelayer init [LOCAL_WEB_SOCKET] [PEGGY_DEPLOYED_ADDRESS] LogLock\(bytes32,address,bytes,address,uint256,uint256\) validator --chain-id=testing
 
 # Enter password and press enter
-# You should see a message like:  Started ethereum websocket... and Subscribed to contract events...
+# You should see a message like: Started ethereum websocket with provider: [LOCAL_WEB_SOCKET] \ Subscribed to contract events on address: [PEGGY_DEPLOYED_ADDRESS]
+
+# The relayer will now watch the contract on Ropsten and create a claim whenever it detects a lock event.
+
 ```
 
-The relayer will now watch the contract on Ropsten and create a claim whenever it detects a lock event.
+### Using Terminal 2: Send lock transaction to contract
 
-## Using the bridge
+```
+# The lock transaction uses the default parameters:
+# [HASHED_COSMOS_RECIPIENT_ADDRESS] = 0x636f736d6f7331706a74677530766175326d35326e72796b64707a74727438383761796b756530687137646668
+# [DEPLOYED_TOKEN_ADDRESS] = 0x0000000000000000000000000000000000000000
+# [WEI_AMOUNT] = 10
+
+$ yarn peggy:lock
+
+# Expected successful output in the relayer console:
+
+New Lock Transaction:
+Tx hash: 0x83e6ee88c20178616e68fee2477d21e84f16dcf6bac892b18b52c000345864c0
+Block number: 5
+Event ID: cc10955295e555130c865949fb1fd48dba592d607ae582b43a2f3f0addce83f2
+Token: 0x0000000000000000000000000000000000000000
+Sender: 0xc230f38FF05860753840e0d7cbC66128ad308B67
+Recipient: cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh
+Value: 10
+Nonce: 1
+
+# You can also confirm the tokens have been minted by using the CLI again:
+$ ebcli query account cosmos1pjtgu0vau2m52nrykdpztrt887aykue0hq7dfh --trust-node
+```
 
-With the application set up and the relayer running, you can now use Peggy by sending a lock transaction to the smart contract. You can do this from any Ethereum wallet/client that supports smart contract transactions.
+We are working on implementing custom parameters to the 'yarn peggy:lock' command. Currently, you can add custom parameters to the lock transaction using an online service such as Remix.
```

Comment:
can you remove this and add this functionality as part of the PR

Reply:
https://github.com/swishlabsco/peggy/pull/2/commits/e9ddf661e1148b851337bd676b62e7247f4f4bd3, https://github.com/swishlabsco/peggy/pull/2/commits/9196e44d8edde051dfcf76d58b58465f2a02d688. Error checking section is due to: https://github.com/trufflesuite/truffle/issues/889. Note acknowledgement of future work for custom parameter validation (js wrapper for cosmos, ethereum address type checking).

Reply:
`yarn peggy:lock` with custom parameters:
<img width="1166" alt="Screen Shot 2019-09-22 at 1 09 39 AM" src="https://user-images.githubusercontent.com/15370712/65384440-5a258b00-dcd7-11e9-83fb-1d59c5ddb160.png">

Output in ebrelayer console:
<img width="619" alt="Screen Shot 2019-09-22 at 1 07 54 AM" src="https://user-images.githubusercontent.com/15370712/65384442-5db91200-dcd7-11e9-935b-de3e7e1a8e44.png">